packages/components/src/number-control/index.tsx

```diff
@@ -88,14 +97,18 @@ export function NumberControl(
 			}
 
 			if ( type === inputControlActionTypes.PRESS_UP ) {
+				// @ts-expect-error TODO: Investigate if this is wrong
 				nextValue = add( nextValue, incrementalValue );
 			}
 
 			if ( type === inputControlActionTypes.PRESS_DOWN ) {
+				// @ts-expect-error TODO: Investigate if this is wrong
 				nextValue = subtract( nextValue, incrementalValue );
 			}
 
+			// @ts-expect-error TODO: Investigate if this is wrong
 			nextState.value = constrainValue(
+				// @ts-expect-error TODO: Investigate if this is wrong
```

Comment:
All of these will mostly depend on the decision that we make regarding the type of the `value` prop